Exadel.Compreface.UnitTests/Services/AbstractBaseServiceTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Exadel.Compreface.Configuration;
+using Flurl.Http.Content;
+using Flurl;
+using Moq;
+using Tynamix.ObjectFiller;
+
+namespace Exadel.Compreface.UnitTests.Services
+{
+    public abstract class AbstractBaseServiceTests
+    {
+        protected ComprefaceConfiguration Configuration { get; }
+
+        protected Mock<IApiClient> ApiClientMock { get; }
+
+        public AbstractBaseServiceTests()
+        {
+            var apiKey = GetRandomString();
+            var baseUrl = GetRandomString();
```

Comment:
BaseUrl was removed. Add, please, domain and port variables and initialize them.